Formula/koka.rb

```diff
@@ -0,0 +1,53 @@
+class Koka < Formula
+  desc "Compiler for the Koka language"
+  homepage "http://koka-lang.org"
+  url "https://github.com/koka-lang/koka/releases/download/v2.3.8/koka-v2.3.9-source.tar.gz"
+  sha256 "ae3e8127b46e51cc4df3525c5f913ecfbaeeb29f0a268e4a3d2e0ac809bc987f"
+  license "Apache-2.0"
+  head "https://github.com/koka-lang/koka.git", branch: "dev"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
```

Comment:
Why is this needed?

Reply:
No idea :-) It's copied from another formula. Shall I remove it?

Reply:
I just pushed an update that removes this.